yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -1252,3 +1252,54 @@ func (yb *TargetYugabyteDB) GetDebeziumValueConverterSuite() map[string]Converte
 func (yb *TargetYugabyteDB) MaxBatchSizeInBytes() int64 {
 	return 200 * 1024 * 1024 // 200 MB
 }
+
+func (yb *TargetYugabyteDB) GetImportedEventsStatsForTable(tableName string, migrationUUID uuid.UUID) (int64, int64, int64, int64, error) {
+	var total_events, num_inserts, num_updates, num_deletes int64
+	if !strings.Contains(tableName, ".") {
+		tableName = yb.getTargetSchemaName(tableName) + "." + tableName
+	}
+	err := yb.connPool.WithConn(func(conn *pgx.Conn) (retry bool, err error) {
+		query := fmt.Sprintf(`SELECT total_events, num_inserts, num_updates, num_deletes FROM %s 
+		WHERE table_name='%s' AND migration_uuid='%s'`, EVENTS_PER_TABLE_METADATA_TABLE_NAME, tableName, migrationUUID)
+		log.Infof("query to get import stats for table %s: %s", tableName, query)
+		err = conn.QueryRow(context.Background(), query).Scan(&total_events, &num_inserts, &num_updates, &num_deletes)
+		return false, err
+	})
+	if err != nil {
+		log.Errorf("error in getting import stats from target db: %v", err)
+		return -1, -1, -1, -1, fmt.Errorf("error in getting import stats from target db: %w", err)
+	}
+	return total_events, num_inserts, num_updates, num_deletes, nil
+}
+
+func (yb *TargetYugabyteDB) GetImportedSnapshotRowCountForTable(tableName string) (int64, error) {
+	var totalRowCount int64
+	schema := yb.getTargetSchemaName(tableName)
+	err := yb.connPool.WithConn(func(conn *pgx.Conn) (retry bool, err error) {
+		query := fmt.Sprintf(`SELECT rows_imported FROM %s where schema_name='%s' AND table_name='%s'`,
```

Comment:
How about `SELECT SUM(rows_imported) ...` ?

Reply:
yeah sure, it will make code simpler.